src/gui/optionsdialog.ui

```diff
@@ -2440,9 +2440,9 @@
              </widget>
             </item>
             <item>
-             <widget class="QGroupBox" name="ratioBox">
+             <widget class="QGroupBox" name="seedingLimitsBox">
               <property name="title">
-               <string>Share Ratio Limiting</string>
+               <string>Seeding Limits</string>
```

Comment:
Do you mind also changing the WebUI? shouldn't be hard.

Reply:
Oops totally forgot that.